js/start.js

```diff
@@ -360,54 +366,135 @@ function start() {
   });
 }
 
-// connect to the API websocket
-// todo: this will be incorporated in the server
-// connection flow
-let socketOpened = false;
-let lostSocketConnectionDialog;
-
-app.apiSocket = new Socket(app.getSocketUrl());
-app.apiSocket.on('open', () => {
-  if (!socketOpened) {
-    socketOpened = true;
-    if (lostSocketConnectionDialog) lostSocketConnectionDialog.remove();
-    lostSocketConnectionDialog = null;
-    start();
+function connectToServer() {
+  const server = app.serverConfigs.activeServer;
+
+  console.log(`Will attempt to connect to server "${server.get('name')}"` +
```

Comment:
I'm leaving this console logging in for now. I'll probably clean it up at some point.